src/com/facebook/buck/apple/project_generator/ProjectGenerator.java

```diff
@@ -1300,6 +1301,18 @@ private PBXNativeTarget generateBinaryTarget(
           appendConfigsBuilder.put("SWIFT_INCLUDE_PATHS", "$BUILT_PRODUCTS_DIR");
         }
 
+        ImmutableList.Builder<String> targetSpecificSwiftFlags = ImmutableList.builder();
+        if (targetNode.getConstructorArg() instanceof SwiftCommonArg) {
+          targetSpecificSwiftFlags.addAll(
+              convertStringWithMacros(
+                  targetNode,
+                  ((SwiftCommonArg) targetNode.getConstructorArg()).getSwiftCompilerFlags()));
+        }
+
+        Iterable<String> otherSwiftFlags =
+            Iterables.concat(
+                swiftBuckConfig.getFlags("compiler_flags").orElse(DEFAULT_SWIFTFLAGS),
```

Comment:
Rather than hardcoding `compiler_flags` here, we should just use `swiftBuckConfig.getFlags()` (which already reads `compiler_flags`).

`SwiftCompile` already uses `getFlags()`, so it's better to be consistent here.